nautobot/core/templates/swagger_ui.js

```
@@ -0,0 +1,123 @@
+  } catch {
+    return false;
+  }
+};
+
+const isSchemaUrl = (url) => {
+  if (!uiInitialized()) {
+    return false;
+  }
+  return url === new URL(ui.getConfigs().url, document.baseURI).href;
+};
+
+const responseInterceptor = (response, ...args) => {
+  if (!response.ok && isSchemaUrl(response.url)) {
+    console.warn("schema request received '" + response.status + "'. disabling credentials for schema till logout.");
+    if (!schemaAuthFailed) {
+      // only retry once to prevent endless loop.
+      schemaAuthFailed = true;
+      setTimeout(() => ui.specActions.download());
+    }
+  }
+  return response;
+};
+
+const injectAuthCredentials = (request) => {
+  let authorized;
+  if (uiInitialized()) {
+    const state = ui.getState().get("auth").get("authorized");
+    if (state !== undefined && Object.keys(state.toJS()).length !== 0) {
+      authorized = state.toJS();
+    }
+  } else if (![undefined, "{}"].includes(localStorage.authorized)) {
+    authorized = JSON.parse(localStorage.authorized);
+  }
+  if (authorized === undefined) {
+    return;
+  }
+  for (const authName of schemaAuthNames) {
+    const authDef = authorized[authName];
+    if (authDef === undefined || authDef.schema === undefined) {
+      continue;
+    }
+    if (authDef.schema.type === "http" && authDef.schema.scheme === "bearer") {
+      request.headers["Authorization"] = "Bearer " + authDef.value;
+      return;
+    } else if (authDef.schema.type === "http" && authDef.schema.scheme === "basic") {
+      request.headers["Authorization"] = "Basic " + btoa(authDef.value.username + ":" + authDef.value.password);
+      return;
+    // This conditional resolves issue #2154 and is the only portion that is updated for this patch.
+    } else if (authDef.schema.type === "apiKey" && authDef.schema.in === "header" && authDef.schema.description === 'Token-based authentication with required prefix "Token"') {
+        request.headers["Authorization"] = "Token " + authDef.value;
```

Can you check `authDef.value` to make sure users haven't already provided `Token ` in their input for backwards compatibility?

And can we strip the cookies from the request so passing a token doesn't silently pass (requests appear to work today with the incorrect value because our API accepts both the session and authorization header).